<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JMainWindow</class>
 <widget class="QMainWindow" name="JMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Journal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	font-family: Roboto, &quot;Noto Sans&quot;, Helvetica, Arial;
}

QMainWindow
{
	background: rgb(245, 245, 245);
}

QPushButton, QTextEdit, QListView, QLineEdit
{
	background: #FFF;
 	border: 1px solid #EDEDEF;
	color: #111;
	padding: .5em 1em;
}

QTextEdit, QLineEdit
{
	border: 1px solid rgb(245, 245, 220) !important;
	background: rgb(255, 255, 240) !important;
	color: #442 !important;
}

QPushButton:hover, QListView::item:hover
{
	background: rgb(250, 250, 250);
}

QPushButton:pressed, QListView::item:selected
{
	border: 1px solid #17F;
	background: #28F;
	color: #FFF;
}

QListView
{
	alternate-background-color: rgb(245, 245, 245);
}

QListView::item
{
	padding: .5em 0;
}

QTextEdit:disabled, QLineEdit:disabled, QPushButton:disabled
{
	background: #EEE;
	color: #555;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addEntryBtn">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeEntryBtn">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveEntryBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="JListView" name="entries">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCalendarWidget" name="datePicker">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Entries</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="entryCountLabel">
         <property name="text">
          <string>0 entries</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QTextEdit" name="entryContent">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="entryTitle">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Entry Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJournal">
    <property name="title">
     <string>Journal</string>
    </property>
    <addaction name="actionNew_Entry"/>
    <addaction name="actionSave_Entry"/>
    <addaction name="actionDelete_Entry"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuJournal"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Entry">
   <property name="text">
    <string>New Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete_Entry">
   <property name="text">
    <string>Delete Entry</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave_Entry">
   <property name="text">
    <string>Save Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>JListView</class>
   <extends>QListView</extends>
   <header>jlistview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>entryTitle</tabstop>
  <tabstop>entryContent</tabstop>
  <tabstop>saveEntryBtn</tabstop>
  <tabstop>addEntryBtn</tabstop>
  <tabstop>entries</tabstop>
  <tabstop>removeEntryBtn</tabstop>
  <tabstop>datePicker</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>JMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Entry</sender>
   <signal>triggered()</signal>
   <receiver>addEntryBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Entry</sender>
   <signal>triggered()</signal>
   <receiver>removeEntryBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Entry</sender>
   <signal>triggered()</signal>
   <receiver>saveEntryBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
